gulpfile.js

```diff
@@ -130,21 +130,21 @@ const prettierTask = function(src, shouldLog) {
     .on('error', onError);
 };
 
-const sasslintTask = function(src, failOnError, shouldLog) {
-  if (shouldLog) {
-    const cmd = `sasslint ${src}`;
-    log('Running', `'${chalk.cyan(cmd)}'...`);
-  }
-  const stream = gulp
-    .src(src)
-    .pipe(sasslint())
-    .pipe(sasslint.format())
-    .pipe(sasslint.failOnError());
-  if (!failOnError) {
-    stream.on('error', onError);
-  }
-  return stream;
-};
+// const sasslintTask = function(src, failOnError, shouldLog) {
+//   if (shouldLog) {
+//     const cmd = `sasslint ${src}`;
+//     log('Running', `'${chalk.cyan(cmd)}'...`);
+//   }
+//   const stream = gulp
+//     .src(src)
+//     .pipe(sasslint())
+//     .pipe(sasslint.format())
+//     .pipe(sasslint.failOnError());
+//   if (!failOnError) {
+//     stream.on('error', onError);
+//   }
+//   return stream;
+// };
```

Comment:
I am trying to figure out why we are commenting this out?

Reply:
Because the linter is broken and unsupported -- we're replacing it in #348, which is a breaking change.